### ISSUE TITLE
Portuguese links in English README.md

### ISSUE THREAD

**Issue by ukoda** [Xinyuan-LilyGO/LilyGo-LoRa-Series#104]
Most of the AliExpress links are to the Portuguese pages, not the English ones and once you land on an AliExpress page in the wrong language it can be hard to change to English, even if you edit the URL by hand.

I would suggest changing links like (https://pt.aliexpress.com/item/32967228739.html) to links like (https://www.aliexpress.com/item/32967228739.html).

**Comment by lewisxhe** [comment]
OK, I will update it, but not now, I will need to update the README again later
